config/initializers/fips.rb

```diff
@@ -0,0 +1,46 @@
+require "openssl"
+require "digest"
+require "digest/md5"
+require "digest/sha1"
+require 'sprockets'
+require 'openid_connect'
+
+# Supress warning messages.
```

Comment:
Not sure how @micahlee feels, but I would strongly suggest heavily commenting this file... maybe even have more comment lines than code ones.  Since it's all configuration, this will be difficult to comprehend in the future.  Why is each constant being changed?  What does that affect and what is the result?

Reply:
Did push a fix.
Please have a look and let know if it is satisfying.